app/src/main/java/org/oppia/app/help/HelpViewModel.kt

```diff
@@ -1,9 +1,24 @@
 package org.oppia.app.help
 
+import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
-import org.oppia.app.fragment.FragmentScope
-import javax.inject.Inject
 
-/** [ViewModel] for [HelpFragment]. */
-@FragmentScope
-class HelpViewModel @Inject constructor() : ViewModel()
+class HelpViewModel : ViewModel{
+  var title = ""
+  constructor() : super()
+  constructor(category: String) : super() {
+    this.title = category
+  }
+
+  var arrayListMutableLiveData = MutableLiveData<ArrayList<HelpViewModel>>()
+  var arrayList = ArrayList<HelpViewModel>()
+
+  fun getArrayList():MutableLiveData<ArrayList<HelpViewModel>>{
+    val category1 = "Frequently Asked Question (FAQs)"
```

Comment:
Never use hardcoded strings, strings should always be defined in `strings.xml` otherwise we won't be able to translate this string to other languages in future.

Anyways your implementation will change based on my previous comment.